front/src/modules/ui/themes/colors.ts

```diff
@@ -60,14 +60,8 @@ export const color = {
   sky20: '#c5fbfb',
   sky10: '#e4fdfd',
   blue: '#1961ed',
-  blue80: '#1a1d2d',
-  blue70: '#1e203f',
-  blue60: '#22244f',
-  blue50: '#2d2e6d',
-  blue40: '#5a5ac0',
-  blue30: '#9797f0',
-  blue20: '#c5c5fb',
-  blue10: '#e4e4fd',
+  lighterBlue: '#f8fafe',
+  lightBlue: '#eef2fd',
```

Comment:
as discussed, we probably need to do something smarted with dark/light mode mapping if the Figma design show different colors